services/cd-service/pkg/cmd/server.go

```diff
@@ -227,6 +235,10 @@ func RunServer() {
 					})
 					api.RegisterBatchServiceServer(srv, &service.BatchServer{
 						Repository: repo,
+						RBACConfig: auth.RBACConfig{
+							DexEnabled: c.DexEnabled,
+							// Todo (BB) pass the rbac config
```

Comment:
TODO comment

Reply:
either handle properly.
Or make a more useful comment like "permission map will be handle in story <story-id>"